### ISSUE TITLE
Icon request: instagram-solid

### ISSUE THREAD

**Issue by pytyl** [FortAwesome/Font-Awesome#14526]
All of the other main social media have a solid version. It is unfortunate that instagram doesn't. I love these social media icons, but the design looks a bit weird when we are forced to use an outline version of the instagram icon. I will attach example of the only instagram icon.

<img width="327" alt="screen shot 2019-01-19 at 12 01 43 pm" src="https://user-images.githubusercontent.com/21296284/51429914-014ea180-1be2-11e9-86be-3f36cae38bd3.png">

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

We pay a lot of attention to brands and we do our best to respect their guidelines.

Actually this is the only official brand for instagram, as per https://en.instagram-brand.com/assets/glyph-icon

You can request a filled in version of the Instagram logo, (it has been already requested here: #13795), but please keep in mind that icons that violate brand guidelines can be removed from the set at any time.

**Comment by curtisday** [comment]
I agree with this. I actually made my own icon that I use but it would be awesome if you guys included it. Even if it's not with the company guidelines, it still looks much better and more consistent with the other logos
<img width="189" alt="Screen Shot 2019-03-29 at 12 25 13 PM" src="https://user-images.githubusercontent.com/49054713/55241362-d904a400-521d-11e9-934b-e2deb55f360a.png">
[instagram_inverse.pdf](https://github.com/FortAwesome/Font-Awesome/files/3023200/instagram_inverse.pdf)

.

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No biggie.
